charts/lotus-bundle/templates/service-monitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.prometheus.serviceMonitor }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ .Release.Name }}-{{ .Values.application.name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      app: {{ .Values.application.name }}
+      release: {{ .Release.Name }}
+  endpoints:
+  - port: {{ .Values.prometheus.port }}
+    path: {{ .Values.prometheus.path }}
+    interval: 30s
+{{- end }}
```

Comment:
Wondering if we should include a service monitor for lotus, don't know what it exports in lite mode, but it would help with at least basic go metrics, and would be beneficial when not running in lite mode.